.github/workflows/wallet-with-console-smoke.yml

```diff
@@ -53,5 +53,5 @@ jobs:
         uses: actions/upload-artifact@v3
         if: always()
         with:
-          name: test screenshots chrome
+          name: test screenshots chrome ${{ matrix.environment }}
```

Comment:
apologies, slightly irrelevant but sneaking this in to catch the console flake. Currently the second lot of screenshots seems to be overwriting the first as both runs had the same artifact name